package.json

```diff
@@ -77,8 +77,7 @@
     "webpack-cli": "3.3.10"
   },
   "dependencies": {
-    "algolia-aerial": "^1.5.3",
```

Comment:
this is used for the documentation website, so it should be a devDependency, not removed. Thanks!